.github/workflows/helmfile_staging_apply.yaml

```diff
@@ -89,6 +89,58 @@ jobs:
           helmfile --environment staging sync
           popd
 
+      - name: Save ENV vars and secrets to AWS Param Store
+        run: |
+          NAMESPACE="notification-canada-ca"
+          get_env_and_secrets() {
```

Comment:
I think this should be a separate script rather than built into GHA - or maybe even part of getContext (getContext --lambda or something)

Reply:
made the update -- if you see any minor issues with the script (like vars out of context or what not) feel free to push an update :)